app/src/main/java/com/voipgrid/vialer/CallActivity.java

```diff
@@ -208,7 +208,7 @@ protected void onResume() {
         refreshAudioSourceButton();
 
         if(!mNetworkUtil.isOnline()) {
-            mNetworkAvailabilityActivity.start();
+          NetworkAvailabilityActivity.start();
```

Comment:
Make sure you've removed the field:

`    NetworkAvailabilityActivity mNetworkAvailabilityActivity = new NetworkAvailabilityActivity();
`